node/src/reactor/main_reactor/upgrading_instruction.rs

```diff
@@ -1,5 +1,9 @@
 use std::time::Duration;
 
+use casper_types::{TimeDiff, Timestamp};
+
+const UPGRADE_TIMEOUT_SECONDS: u32 = 30;
```

Comment:
We should probably move this to the config so that we can adjust this to specific validator setups easily.

Reply:
Added in https://github.com/casper-network/casper-node/pull/3971/commits/18bce1adea6345647e1568f2068b7ddc74c5ed1f